stage0/run.go

```diff
@@ -628,6 +630,36 @@ func prepareStage1Image(cfg PrepareConfig, img types.Hash, cdir string, useOverl
 	return nil
 }
 
+// setJournalPermissions sets ACLs and permissions so the rkt group can access
+// the pod's logs
+func setJournalPermissions(cfg RunConfig, cdir string) error {
+	s1 := common.Stage1ImagePath(cdir)
+
+	journalPath := filepath.Join(s1, "rootfs", "var", "log", "journal")
+	if err := os.MkdirAll(journalPath, os.FileMode(0755)); err != nil {
+		return fmt.Errorf("error creating journal dir: %v", err)
+	}
+	if err := os.Chmod(journalPath, os.FileMode(0755|os.ModeSetgid)); err != nil {
+		return err
+	}
+
+	a, err := acl.Parse(fmt.Sprintf("u::rwx,g::r-x,g:%d:r-x,o:r-x,m:r-x", cfg.RktGid))
```

Comment:
I have no clue about Posix ACL, but this line looks like you are duplicating the chmod call above.